src/blueprint/into_schema.rs

```diff
@@ -70,12 +71,17 @@ fn to_type(def: &Definition) -> dynamic::Type {
 
                                         let const_value =
                                             expr.eval(ctx, &Concurrent::Sequential).await?;
-
                                         let p = match const_value {
-                                            ConstValue::List(a) => FieldValue::list(a),
-                                            a => FieldValue::from(a),
+                                            ConstValue::List(a) => Some(FieldValue::list(a)),
+                                            a => {
+                                                if a == ConstValue::Null && is_list {
```

Comment:
why return none only for list? 

I think it should return none if it's any null

Reply:
In case of object you need to return None on the field